odoo/tools/sql.py

```diff
@@ -249,7 +249,7 @@ def drop_view_if_exists(cr, viewname):
         cr.execute("DROP MATERIALIZED VIEW {} CASCADE".format(viewname))
 
 def escape_psql(to_escape):
-    return to_escape.replace('\\', r'\\').replace('%', '\%').replace('_', '\_')
+    return to_escape.replace('\\', r'\\').replace('%', r'\%').replace('_', r'\_')
```

Comment:
```suggestion
    return to_escape.replace('\\', r'\\').replace('%', '\\%').replace('_', '\\_')
```
To confirm but same as above

Reply:
I'm sorry but I wont handfix that in all versions, if the script behaviour is correct I'll do the automated way in all versions. 
I can wait a little if you want to fix your cases by hand before I start merging this pr.